docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_7.rst

```diff
@@ -50,22 +50,23 @@ Release notes: Open GEE 5.3.7
          * - Number
            - Description
            - Resolution
-         * - 9999
-           - Add description
-           - Add resolution
+         * - 1868
+           - Apache proxy configurations missing in HTTP modules
+           - Added missing apache module for gehttpd server build
+         * - 1861
+           - Invalid variables when reporting postgres errors
+           - Corrected variable name
```

Comment:
Minor: multiple variables were fixed, so technically it's "Corrected variable names", but I'm not sure that's worth worrying about.